sos/utilities.py

```diff
@@ -147,7 +147,7 @@ def _child_prep_fn():
         else:
             expanded_args.append(arg)
     try:
-        p = Popen(expanded_args, shell=False, stdout=PIPE,
+        p = Popen(expanded_args, stdin=PIPE, shell=False, stdout=PIPE,
```

Comment:
Why this change, please?

@bmr-cymru and @TurboTurtle , any comment here? Could it have some drawbacks?

Reply:
Thanks a lot for taking a look.
Without that change, podman hangs for me [0] and with strace I could see that the process is receiving SIGTTIN all the time.

[0] https://bugzilla.redhat.com/show_bug.cgi?id=1732525